src/neo4j_genai/indexes.py

```diff
@@ -119,7 +123,9 @@ def create_fulltext_index(
         logger.info(f"Creating fulltext index named '{name}'")
         driver.execute_query(query, {"name": name})
     except neo4j.exceptions.ClientError as e:
-        raise Neo4jIndexError(f"Neo4j fulltext index creation failed {e}")
+        raise Neo4jIndexError(
+            f"Neo4j fulltext index creation failed {e.message}"
```

Comment:
How do you determine whether to print e.message vs e.errors()?

Reply:
`e.message` for Neo4j errors, `e.errors()` for validation errors